### ISSUE TITLE
Review from Emily

### ISSUE THREAD

**Issue by Emily-Jiang** [OpenLiberty/draft-guide-istio#9]
1. A step to clone the repo is missing and it directly mentions to navigate to finish folder

**Comment by Emily-Jiang** [comment]
3. It did not run against Istio 1.0
when trying to deploy the image to Istio. I got this:
Error: failed to convert to proto. unknown field "sdsRefreshDelay" in v1alpha1.MeshConfig
error: no objects passed to apply

**Comment by Emily-Jiang** [comment]
for 2) I think I know what is wrong. I assumed the test will run with `mvn clean install`. When I finish reading it, I realised the host and port need to be passed in. You can easily add a null check below and print out a better error message if they are null. I think I am not the only one who assume `mvn clean install `works without supplying any additional parameters.
`public static void oneTimeSetup() {
        testIp = System.getProperty("test.ip");
        testPort = System.getProperty("test.port");
        sysUrl = "http://" + testIp + ":" + testPort + "/";
        invUrl = "http://" + testIp + ":" + testPort + "/";
    }`

**Comment by Emily-Jiang** [comment]
@proubatsis did you see my suggestion to improve oneTimeSetup to do a null check?

**Comment by proubatsis** [comment]
@Emily-Jiang Yes, I saw the suggestion, but I am a bit unclear on the specifics. Are you saying that the tests should still fail but with a better error message? Or should they pass if the host and port are null?

**Comment by proubatsis** [comment]
@Emily-Jiang The null check has been incorporated into #8
